include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2642,7 +2665,34 @@ class FEFaceEvaluation : public FEEvaluationAccess<dim,
    */
   void
   integrate(const EvaluationFlags::EvaluationFlags integration_flag,
-            VectorizedArrayType                   *values_array);
+            VectorizedArrayType                   *values_array,
+            const bool                             sum_into_values = false);
+
+  /**
+   * This function tests the values, gradients and Hessians submitted on the
+   * face quadrature points by multiplying with the in-face basis function
+   * values, gradients and Hessians and accumulating to the respective face
+   * DoFs.
+   */
+  void
+  integrate_in_face(const EvaluationFlags::EvaluationFlags integration_flag);
+
+  /**
+   * Collects the contributions from the face DoFs of values, normal gradients
+   * and normal Hessians to the internal cell DoFs.
+   */
+  void
+  collect_from_face(const EvaluationFlags::EvaluationFlags integration_flag,
```

Comment:
Everyone fine with the name of the new function?

Reply:
I am open to suggestions! It sounded like the natural transpose to `project_to_face` to me.